Document/0x05h-Testing-Platform-Interaction.md

```diff
@@ -703,9 +703,9 @@ dz> run app.activity.start  --action android.intent.action.VIEW --data-uri "sms:
 
 ### Overview
 
-With [Google Play Instant](https://developer.android.com/topic/google-play-instant/overview "Google Play Instant") you can now create Instant apps. An instant apps can be instantly launched from a browser or the "try now" button from the app store from Android 6.0 (API level 23) onward. They do not require any form of installation. There are a few challenges with an instant app:
+With [Google Play Instant](https://developer.android.com/topic/google-play-instant/overview "Google Play Instant") you can now create Instant apps. An instant apps can be instantly launched from a browser or the "try now" button from the app store from Android 5.0 (API level 21) onward. They do not require any form of installation. There are a few challenges with an instant app:
 
-- There is a limited amount of size you can have with an instant app (max 10 mb).
+- There is a limited amount of size you can have with an instant app (max 15 mb).
```

Comment:
```suggestion
- There is a limited amount of size you can have with an instant app.
```

Reply:
I'd remove it because it doesn't really add value and this way we don't have to maintain it.